typed/parser_test.go

```diff
@@ -99,7 +97,7 @@ func BenchmarkConvertUnstructured(b *testing.B) {
 				b.ResetTimer()
 
 				for n := 0; n < b.N; n++ {
-					_ = u.AsValue().ToUnstructured(false)
+					// This is a no-op now.
```

Comment:
might as well delete the whole for loop, this comment will be super mysterious to someone not looking at the diff.

Reply:
I've removed the loop and clarified the comment. I've kept the dummy benchmark though.